From 5205c73d9311bcf458ed0fe1f4ce381bf3842b3d Mon Sep 17 00:00:00 2001
From: Herwig Hochleitner <herwig@bendlas.net>
Date: Thu, 29 Jun 2023 18:26:51 +0200
Subject: [PATCH] nixos/gitea: add WORK_PATH to config

this is in preparation for 1.20, which needs this option set
---
 nixos/modules/services/misc/gitea.nix | 1 +
 1 file changed, 1 insertion(+)

diff --git a/nixos/modules/services/misc/gitea.nix b/nixos/modules/services/misc/gitea.nix
index 72fad73bae509..7e7ac355edda9 100644
--- a/nixos/modules/services/misc/gitea.nix
+++ b/nixos/modules/services/misc/gitea.nix
@@ -15,6 +15,7 @@ let
     APP_NAME = ${cfg.appName}
     RUN_USER = ${cfg.user}
     RUN_MODE = prod
+    WORK_PATH = ${cfg.stateDir}
 
     ${generators.toINI {} cfg.settings}
 
